### DISCUSSION TITLE
ECS containers should be limited to read-only access to root filesystems

### DISCUSSION THREAD

**Discussion by andrea-avogadro-archiva** [aws/copilot-cli#5774] [Q&A]
Hi,
Security Hub tells me that a task created with Copilt do not comply with the following rule: https://docs.aws.amazon.com/securityhub/latest/userguide/ecs-controls.html#ecs-5

It seems not possible to set this option via Service Manifest, am I right?

Maybe could be useful to add this option.

Thanks
Andrea

**Comment by iamhopaul123** [comment]
Hello @andrea-avogadro-archiva. You can use [`copilot svc override`](https://aws.github.io/copilot-cli/docs/developing/overrides/yamlpatch/) to override and set any field in your task definition!
